packages/output-handler/src/index.ts

```diff
@@ -33,6 +39,14 @@ const messageBody = (
 	`;
 };
 
+const failureMessageBody = (originalFilename: string): string => {
+	return `
+		<h1>Transcription for ${originalFilename} has failed 😢</h1>
```

Comment:
too jaunty?

Reply:
Ah I love it but my suspicion might is editorial might not love it